deploy/manifests/dev/us-east-2/tenant/storetheindex/instances/kustomization.yaml

```diff
@@ -9,3 +9,6 @@ resources:
   - ber
   - cali
   - ago
+  - alva
+  - bria
+  - cora
```

Comment:
Postpone running them until they are configured to talk to a dhstore instance backed by FDB?
```suggestion

```

Reply:
They are all configured to use an assigner, but are not in the assigner's pool so will not be assigned anything. This means that they will not act on announcements received. So, running is OK and they will do nothing until the assigner is configured to assign to them.

We can certainly postpone running them until everything is ready. So I have changed to config as per above.

Reply:
ty